src/app/zap-templates/templates/app/helper.js

```diff
@@ -195,29 +195,40 @@ function chip_endpoint_generated_functions()
 function chip_endpoint_generated_commands_list(options)
 {
   let ret = [];
+  let index = 0;
```

Comment:
Note: This PR depends on https://github.com/project-chip/zap/pull/480, this comment will block the PR from merging accidentally.

Reply:
Zap synced in #18134